### ISSUE TITLE
Error Response for DRM

### ISSUE THREAD

**Issue by Hendrikv1990** [an-rahulpandey/cordova-plugin-mediapicker#17]
When an item is not downloaded from iTunes on the Phone, the songurl returns nil. But the Error message returns "This song is protected by Digital Rights Management (DRM) and cannot be accessed." which is then not accurate.

After it's actually downloaded, the error disappears. Although left already in a confused state about the DRM error.. ( code below )

Is it possible to identify the DRM in a different way or somehow adjust this if statement? Would greatly appreciate it since we would otherwise overwrite this string in our code :)

Thanks

`if(!songurl){
               plresult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@“This song is protected by Digital Rights Management (DRM) and cannot be accessed.“];
               [self.commandDelegate sendPluginResult:plresult callbackId:callbackID];
               break;
           }`

**Comment by jonbongato** [comment]
@Hendrikv1990 Hi Hendrik, how did you go about this? I've been encountering same issue.

**Comment by an-rahulpandey** [comment]
@Hendrikv1990 @jonbongato Can you paste the javascript code, to see how you are calling the plugin?

**Comment by Hendrikv1990** [comment]
Yes let me take a look. I didn't solve it until now. Someone suggested to do a replace string..

@an-rahulpandey here it is.

`const deferred = this.$q.defer();

    if (window.plugins) {
      window.plugins.mediapicker.getAudio(success => {
        deferred.resolve(decodeURI(success[0].exportedurl.replace("file:///","/")));
        console.log('Successfully got the audio', success);
      }, error => {
        deferred.reject(error);
        console.log('Error picking audio', error);
        this.$ionicPopup.alert({title: error});
      }, false, true);
    } else {
      deferred.resolve(SAMPLE_TRACK);
      this.fakeTrack = true;
    }

    return deferred.promise;`

**Comment by an-rahulpandey** [comment]
Hi,
I am still trying to see if their is a better way to identify DRM content, I am not able to find anything yet. 
You can also use `cordova.file.documentsDirectory` to directly access the documents directory

**Comment by Hendrikv1990** [comment]
Ok thanks for your help.. Currently we just tell people to download the songs on their devices before using the app. But not that user friendly.
